uaclient/cli.py

```diff
@@ -1553,7 +1553,14 @@ def action_status(args, *, cfg):
     show_beta = args.all if args else False
     token = args.simulate_with_token if args else None
     active_value = ua_status.UserFacingConfigStatus.ACTIVE.value
-
+    try:
+        if contract.is_contract_changed(cfg):
+            cfg.add_notice("", ua_status.NOTICE_REFRESH_CONTRACT_WARNING)
```

Comment:
Let's add this notice on the messages module, since we already have a PR(#2023) that does that

Reply:
Alright.